dev/devicelab/manifest.yaml

```diff
@@ -566,6 +566,14 @@ tasks:
     stage: devicelab
     required_agent_capabilities: ["mac-catalina/android"]
 
+  # macOS target platform tests
+  smoke_catalina_hot_mode_dev_cycle_macos_target_benchmark:
+    description: >
+      Checks the functionality and performance of hot reload on a macOS target platform
+    stage: devicelab
+    required_agent_capabilities: ["mac/ios"]
```

Comment:
Is it possible to also add `"mac/android", "mac/ios32", "mac/iphonexs"` since this could run on any of those?  That would give the test a wider choice of machines to run on and allow more throughput.

Ideally, the devicelab would be able to just take "mac" as a required agent capability.

Reply:
I think the match logic is "all of" and not "one of", but it should probably be updated to account for this